doc/sphinx/Pacemaker_Python_API/index.rst

```diff
@@ -1,11 +1,20 @@
-Contents
---------
+Pacemaker Python Module
```

Comment:
I really don't understand what this commit is doing besides dropping the two templates and changing the title of this page. So, specifically the new `:maxdepth:` and `Index -> :ref:` lines.

I also don't understand what in this commit is taking the place of the templates that we're dropping.

Reply:
Ah, okay. I'm not immediately familiar with these directives, and these didn't seem related to the commit message at all. Probably fine.